server/sonar-server/src/main/java/org/sonar/server/setting/ws/ScannerSettings.java

```diff
@@ -39,31 +39,25 @@
 /**
  * This class returns the list of settings required on scanner side (licenses, license hashes, server ids, etc.)
  */
-public class ScannerSettings implements Startable {
+public class ScannerSettings {
 
-  private static final String SONAR_PREFIX = "sonar.";
   private static final Set<String> SERVER_SETTING_KEYS = ImmutableSet.of(PERMANENT_SERVER_ID, SERVER_STARTTIME, SERVER_ID);
 
+  private final DbClient dbClient;
   private final PropertyDefinitions propertyDefinitions;
-  private final PluginRepository pluginRepository;
 
-  private Set<String> scannerSettingKeys;
-
-  public ScannerSettings(PropertyDefinitions propertyDefinitions, PluginRepository pluginRepository) {
+  public ScannerSettings(DbClient dbClient, PropertyDefinitions propertyDefinitions) {
+    this.dbClient = dbClient;
     this.propertyDefinitions = propertyDefinitions;
-    this.pluginRepository = pluginRepository;
   }
 
-  @Override
```

Comment:
It was a big mistake to load settings at startup, as settings value can change at any time !
I'll try to add ITs BTW to cover this.